torch_ort_inference/docker/Dockerfile.ort-infer-stable-torch1120-onnxruntime1120-openvino-ubuntu18.04

```diff
@@ -0,0 +1,85 @@
+#-------------------------------------------------------------------------
+# Copyright(C) 2022 Intel Corporation.
+# SPDX-License-Identifier: MIT
+#--------------------------------------------------------------------------
+
+ARG OPENVINO_VERSION="2022.1.0"
+ARG OVPT_BRANCH="releases/v0.5"
+
+FROM openvino/ubuntu18_runtime:${OPENVINO_VERSION}
+
+LABEL description="This is the runtime image for OpenVINO™ integration with PyTorch on Ubuntu 18.04 LTS"
+
+LABEL vendor="Intel Corporation"
+
+USER root
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+SHELL ["/bin/bash", "-xo", "pipefail", "-c"]
+
+RUN apt-get update; \
+    apt-get install -y --no-install-recommends \
+    git wget build-essential \
+    python3.8 python3.8-venv python3-pip python3-dev libpython-dev libpython3.8-dev; \
+    rm -rf /var/lib/apt/lists/*;
+
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1; \
+    update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 2;
+
+WORKDIR /home/ovtort/
+
+RUN git clone https://github.com/intel-staging/ort.git && \
+    cd ort/torch_ort_inference && \
+    git checkout ${OVPT_BRANCH} && \
+    git submodule init && \
+    git submodule update --recursive
+
+RUN python3 -m pip install --upgrade pip && pip install -U numpy && pip install wheel
+
+ENV WORKDIR_PATH ort/torch_ort_inference
+WORKDIR ${WORKDIR_PATH}
+
+RUN pip install onnxruntime-openvino>=1.12.0
+
+RUN python setup.py install && \
+    python build.py && \
+    cd dist/ && \
+    pip install *.whl
+
+RUN pip install wget pandas torchvision Pillow transformers
```

Comment:
Duplicate wget install

Reply:
fixed, here we are using pip based wget against the apt based wget. Removed apt based wget